diff --git a/src/drivers/drivers.wxs b/src/drivers/drivers.wxs
index 5a92655..753914d 100644
--- a/src/drivers/drivers.wxs
+++ b/src/drivers/drivers.wxs
@@ -74,7 +74,7 @@
 	      <File Id='XenIfaceSys' Name='XenIface.sys' DiskId='1' Source='xeniface\$(var.build)\xeniface.sys' KeyPath='yes' />
           <File Id='XenIfaceInf' Name='XenIface.inf' DiskId='1' Source='xeniface\$(var.build)\xeniface.inf' />
           <File Id='XenIfaceCat' Name='XenIface.cat' DiskId='1' Source='xeniface\$(var.build)\xeniface.cat' />
-          <File Id='xeniface_coinst' Name='xeniface_coinst.dll' DiskId='1' Source='xeniface\$(var.build)\xeniface_coinst.dll' /> 
+<!--          <File Id='xeniface_coinst' Name='xeniface_coinst.dll' DiskId='1' Source='xeniface\$(var.build)\xeniface_coinst.dll' />  -->
           <File Id='LiteAgentExe' Name='LiteAgent.exe' DiskId='1' Source='xeniface\$(var.build)\LiteAgent.exe' />
           <difx:Driver PlugAndPlayPrompt='no' DeleteFiles='no' ForceInstall='no' Sequence='2' Legacy="yes"/>
         </Component>
@@ -95,7 +95,7 @@
             <File Id='XenVbdSys' Name='XenVbd.sys' DiskId='1' Source='xenvbd\$(var.build)\xenvbd.sys' KeyPath='yes' />
             <File Id='XenVbdCat' Name='XenVbd.cat' DiskId='1' Source='xenvbd\$(var.build)\xenvbd.cat' />
             <File Id='XenCrshSys' Name='XenCrsh.sys' DiskId='1' Source='xenvbd\$(var.build)\xencrsh.sys' />
-            <File Id='XenDiskSys' Name='XenDisk.sys' DiskId='1' Source='xenvbd\$(var.build)\xendisk.sys' />
+<!--            <File Id='XenDiskSys' Name='XenDisk.sys' DiskId='1' Source='xenvbd\$(var.build)\xendisk.sys' /> -->
             <File Id='XenVbdCoInstDll' Name='XenVbd_coinst.dll' DiskId='1' Source='xenvbd\$(var.build)\xenvbd_coinst.dll' />
             <File Id='XenVbdInf' Name='XenVbd.inf' DiskId='1' Source='xenvbd\$(var.build)\xenvbd.inf' />
             <difx:Driver PlugAndPlayPrompt='no' DeleteFiles='no' ForceInstall='no' Sequence='3' Legacy="yes"/>
